drivers/coreaudio/audio_driver_coreaudio.cpp

```diff
@@ -243,25 +243,47 @@ OSStatus AudioDriverCoreAudio::input_callback(void *inRefCon,
 }
 
 void AudioDriverCoreAudio::start() {
-	if (!active) {
+	if (!active && !sleeping) {
```

Comment:
I think checking `active` first like this `if (active) { return; }` to perform early exit and then checking `sleeping` should make it a bit more readable.

Reply:
This complicated the diffs a bit for anyone reviewing, but I agree it's more readable when viewed on its own. Fixed.